Allow enabling currency-mismatched payment methods when Adaptive Pricing is enabled (#5199)
woocommerce/woocommerce-gateway-stripe: changelog.txt

*** Changelog ***

= 10.6.0 - xxxx-xx-xx =

> * Add - Allow enabling payment methods in Stripe settings when the store currency does not match, if Adaptive Pricing is enabled

Reviewer: Not blocking, but I think we can simplify this a bit.
```suggestion
* Add - Allow payment methods for other currencies to be enabled when Adaptive Pricing is enabled
```
